bin/generate_practice_exercise.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Exit if anything fails.
+set -euo pipefail
+
+# If argument not provided, print usage and exit
+if [ -z "$1" ]; then
+    echo "Usage: bin/generate.sh <exercise-slug>"
+    exit 1
+fi
+
+SLUG="$1"
+exercise_dir="exercises/practice/${SLUG}"
+
+# build configlet
+echo "Fetching latest version of configlet..."
+./bin/fetch-configlet
+
+# Preparing config.json
+echo "Please adapt and add the following in config.json before continuing: "
+UUID=$(bin/configlet uuid)
+jq -n --arg slug "$SLUG" --arg uuid "$UUID" \
```

Comment:
You could use `jq` to add the new entry to the `config.json` file. See https://stackoverflow.com/a/42248841

Reply:
That seems obvious, I don't understand why I haven't done that already :)

Reply:
Ah, I remember now. It's a pain to do in-place replacement, it's not a feature of `jq`. I'll figure it out :)